app/src/main/res/values/strings.xml

```diff
@@ -43,6 +44,26 @@
         <item>CAD</item>
     </string-array>
 
+    <string-array name="weight">
```

Comment:
Im not too sure on this but I believe a more appropriate place for these would be in arrays.xml. Up to you, either works.

Reply:
The list for the currency converter spinner was also here so I put mine here as well.

Reply:
As per [Android's documentation](https://developer.android.com/guide/topics/resources/string-resource.html#StringArray), there is no need to create a new arrays.xml file to store String Arrays as they are still String ressources.